.circleci/config.yml

```diff
@@ -53,16 +53,48 @@ jobs:
           command: |
             . venv/bin/activate
             cd python
-            python3 -m pytest --html=pytest_report.html
-          no_output_timeout: 30m  # Time for running the tests before we timeout
+            python3 -m pytest ml4ir/applications/classification/tests --html=pytest_report.html
+          no_output_timeout: 45m  # Time for running the tests before we timeout
```

Comment:
Ouch, we can pass +30m for the tests? Could be interesting to see which test run too long. cc @lastmansleeping

Reply:
In some cases I went up to ~25. Moved this to ensure it is not a blocker.

Reply:
@balikasg @Ullimague I don't think we should do that. Let's identify the tests taking too long and fix them. Tests should take a handful of minutes.